### DISCUSSION TITLE
Unable to receive verification code via sms

### DISCUSSION THREAD

**Discussion by quynb93** [vercel/community#4018] [Account Issue] [answered]
### Question

I still have an issue can not sign-in to Vercel with my account, In the verification step my phone number has been blocked for 12 hours, but it's 24 hours since the last login. but I still can not verify.

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble logging in. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)

To open a ticket with the appropriate team, please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy.
